LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2019 Kontrol SAS
```

Comment:
not 100% sure about this one. But this was the right license at the time the lib was published, so, good enough for me

Reply:
Not sure either. If it's anything like a website, it should be the current year of even `2019-2023`, which means we would need to update all licenses... yay.

Reply:
https://opensource.stackexchange.com/questions/2171/refreshing-shared-copyright-dates-in-apache-2-0-licensed-code

Looks like we're good, no need to update the date.

Reply:
My comment was more about `Kontrol SAS`

Reply:
Oh...

I changed it to `2023 Doctolib`, no need to make this more complicated than it needs to.

Absence of license file means that no one can do anything with the code, so we're covered in the past. We _could_ use an anterior date and open-up the source prior to adding the license file, that actually works, but there's no point AFAIK.

As for the present, we could still use `Kontrol SAS` and it would translate to `Tanker SAS` and then `Doctolib` just fine, but since we only saw the issue now there doesn't seem to be any point to that stunt either.

So let's keep it simple!